sphinxext/rediraffe.py

```diff
@@ -379,38 +379,40 @@ def abs_path_in_src_dir_w_src_suffix(filename: str) -> Union[Path, None]:
                     f"deleted file {deleted_file} redirects to {absolute_redirects[deleted_file]}."
                 )
             else:
-                err_msg = f'{red("(broken)")} {deleted_file} was deleted but is not redirected!'
+                err_msg = f"{red('(broken)')} {deleted_file} was deleted but is not redirected!"
                 logger.error(err_msg)
                 self.app.statuscode = 1
 
-        with redirects_path.open("a") as redirects_file:
```

Comment:
This looks like a big change, but all I did was move the file open to wrap the one line that needs it, then had to re-indent a large block.